store/sqlstore/oauth_store.go

```diff
@@ -53,12 +53,6 @@ func newSqlOAuthStore(sqlStore *SqlStore) store.OAuthStore {
 	return as
 }
 
-func (as SqlOAuthStore) createIndexesIfNotExists() {
-	as.CreateIndexIfNotExists("idx_oauthapps_creator_id", "OAuthApps", "CreatorId")
-	as.CreateIndexIfNotExists("idx_oauthaccessdata_user_id", "OAuthAccessData", "UserId")
-	as.CreateIndexIfNotExists("idx_oauthaccessdata_refresh_token", "OAuthAccessData", "RefreshToken")
```

Comment:
We are removing stuff for `OAuthAccessData` but the PR merely focuses on `OAuthApps`.

Reply:
These are forgotten
- https://github.com/mattermost/mattermost-server/blob/6ac9cdc98767dc7adaa7dc8b2792c9d2c77f5c0f/db/migrations/mysql/000025_create_oauth_access_data.up.sql#L49
- https://github.com/mattermost/mattermost-server/blob/6ac9cdc98767dc7adaa7dc8b2792c9d2c77f5c0f/db/migrations/mysql/000025_create_oauth_access_data.up.sql#L139